src/AppInstallerCommonCore/Errors.cpp

```diff
@@ -67,6 +67,8 @@ namespace AppInstaller
                 return "Command requires administrator privileges to run";
             case APPINSTALLER_CLI_ERROR_SOURCE_NOT_SECURE:
                 return "The source location is not secure";
+            case APPINSTALLER_CLI_ERROR_SOURCE_NOT_REMOTE:
+                return "The source location is not remote";

```

Comment:
Nit: the ordering of these follows order of declarations in AppInstallerErrors.h
Also by the way, can you help add  APPINSTALLER_CLI_ERROR_JSON_INVALID_FILE  which we somehow failed previously?

Reply:
Add the error in order and also added APPINSTALLER_CLI_ERROR_JSON_INVALID_FILE.